microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    node_requirement,
+    schema,
+    search_space,
+)
+from lisa.tools import Lscpu
+from lisa.util import SkippedException
+from microsoft.testsuites.cloud_hypervisor.ch_tests_tool import CloudHypervisorTests
+
+
+@TestSuiteMetadata(
+    area="cloud-hypervisor",
+    category="community",
+    description="""
+    This test suite is for executing the tests maintained in the
```

Comment:
What happened, if we run those tests on Azure?

Reply:
They will pass. The reason for adding these tests is so that we can run them on Azure VMs.